resources/build/increment-version.sh

```diff
@@ -156,6 +156,37 @@ if [ "$action" == "commit" ]; then
   git checkout "$branch"
   hub pull-request -f --no-edit -b $base -l auto
 
+  #
+  # If we are on a stable-x.y branch, then we also want to merge changes to
+  # HISTORY.md to master. We don't want to do this for beta or alpha builds;
+  # beta changes will be merged to master periodically anyway.
+  #
+  if [[ "$base" =~ ^stable-[0-9]+\.[0-9]+$ ]]; then
+    git switch master
+
+    # In order to avoid potential git conflicts, we run the history collater
+    # again on the master HISTORY.md. Note that the script always exits 1 to
+    # indicate it hasn't updated VERSION.md. We could tweak that in the future.
+    node resources/build/version/lib/index.js history --no-write-github-comment -t "$GITHUB_TOKEN" -b "$base" || true
```

Comment:
Is `--no-write-github-comment` equivalent to `-write-github-comment false`?

Reply:
[Yes](https://github.com/yargs/yargs/blob/HEAD/docs/api.md#booleankey).